go.sum

```diff
@@ -644,6 +644,7 @@ github.com/prometheus/procfs v0.0.2/go.mod h1:TjEm7ze935MbeOT/UhFTIMYKhuLP4wbCsT
 github.com/prometheus/procfs v0.0.3/go.mod h1:4A/X28fw3Fc593LaREMrKMqOKvUAntwMDaekg4FpcdQ=
 github.com/prometheus/procfs v0.0.10 h1:QJQN3jYQhkamO4mhfUWqdDH2asK7ONOI9MTWjyAxNKM=
 github.com/prometheus/procfs v0.0.10/go.mod h1:7Qr8sr6344vo1JqZ6HhLceV9o3AJ1Ff+GxbHq6oeK9A=
+github.com/protocolbuffers/protobuf v3.14.0+incompatible h1:8r0H76h/Q/lEnFFY60AuM23NOnaDMi6bd7zuboSYM+o=
```

Comment:
(i *think* this should disappear since you removed the examples directory, if not curious about why this got added. what's the output of `go mod why github.com/protocolbuffers/protobuf`)?

Reply:
Ah, that's weird. Yeah, if I delete this `make vendor_rebuild` doesn't put it back I guess? I'll delete this line, re-push, and make sure it goes green again.

Reply:
`go mod tidy` should clear it.